go/vt/vtgate/vtgate.go

```diff
@@ -60,6 +61,7 @@ var (
 	_                  = flag.Bool("disable_local_gateway", false, "deprecated: if specified, this process will not route any queries to local tablets in the local cell")
 	maxMemoryRows      = flag.Int("max_memory_rows", 300000, "Maximum number of rows that will be held in memory for intermediate results as well as the final result.")
 	warnMemoryRows     = flag.Int("warn_memory_rows", 30000, "Warning threshold for in-memory results. A row count higher than this amount will cause the VtGateWarnings.ResultsExceeded counter to be incremented.")
+	defaultDDLStrategy = flag.String("default-ddl-strategy", "", "Set default strategy for DDL statements. Override with @@ddl_strategy session variable")
```

Comment:
I think we can drop the word `default` from here. This is how some of the other startup parameters behave.

Reply:
fixed